openapi.yaml

```
@@ -27258,6 +27668,130 @@ components:
           x-linode-cli-display: 6
           example: '2018-01-01T13:46:32'
           readOnly: true
+    PlacementGroup:
+      type: object
+      properties:
+        id:
+          type: integer
+          nullable: false
+          description: >
+            The placement group's ID. You need to provide it for all operations impacting it.
+          example: 528
```

To ensure the ID is always displayed in the output table, we can use the `x-linode-cli-display` extension here:

```suggestion
          example: 528
          x-linode-cli-display: 1
```